Sources/MongoSwift/MongoCollection+Read.swift

```diff
@@ -185,7 +194,8 @@ public struct AggregateOptions: Encodable {
     public let readConcern: ReadConcern?
 
     /// A ReadPreference to use for this operation.
-    public let readPreference: ReadPreference?
+    // swiftlint:disable:next redundant_optional_initialization
```

Comment:
you might want to check if adding this intermediate comment will prevent the docstring from being associated with the property (if you haven't already)

Reply:
good catch, they weren't being documented properly. fixed.